number_parser_data/supplementary_translation_data/en.json

```diff
@@ -0,0 +1,10 @@
+{
+    "UNIT_NUMBERS": {},
+    "BASE_NUMBERS": {},
+    "MTENS": {},
+    "MHUNDREDS": {},
+    "MULTIPLIERS": {},
+    "VALID_TOKENS": {
+        "tokens":["and","-"]
+    }
+}
```

Comment:
We will need to add the plural forms for "hundred" ("hundreds"), "thousand", "million", "billion", "trillion" and "quadrillion" as well as adding some other equivalences. 

For this reason, I'm wondering if the result "data" file should be composed by regexes instead of single words (to allow to match "hundred" with an optional trailing "s". We can continue using the current approach (single word) but if we see a lot of almost duplicated words, we can think about changing the approach.